### PR TITLE
[翻译完成] flutter.cn/src/docs/cookbook/navigation/returning-data.md

### PR DESCRIPTION
**Pull request by songfei**
翻译完成 returning-data.md

完成 Issue #128

### CODE REVIEW

File: src/docs/cookbook/navigation/returning-data.md [hunk 1]
```diff
@@ -16,22 +16,46 @@ we push a new screen that presents two options to a user. When the user taps on
 an option, we'll want to inform our first screen of the user's selection so it
 can act on that information!
 
+在某些场景，我们需要在回退到上一屏的时候，同时返回一些数据。比如，我们跳转到新的一屏，有两个选项让用户选择，当用户点击某个选项后会返回到第一屏，同时在第一屏可以知道用户选择的信息。
```
Comment (chenglu) [inline src/docs/cookbook/navigation/returning-data.md]:
```suggestion
在某些场景下，我们需要在回退到上一屏时同时返回一些数据。比如，我们跳转到新的一屏，有两个选项让用户选择，当用户点击某个选项后会返回到第一屏，同时在第一屏可以知道用户选择的信息。
```

File: src/docs/cookbook/navigation/returning-data.md [hunk 2]
```diff
@@ -16,22 +16,46 @@ we push a new screen that presents two options to a user. When the user taps on
...
+
 How can we achieve this? Using
 [`Navigator.pop`]({{site.api}}/flutter/widgets/Navigator/pop.html)!
 
+要怎么做呢？ 使用 [`Navigator.pop`]({{site.api}}/flutter/widgets/Navigator/pop.html) ！
```
Comment (chenglu) [inline src/docs/cookbook/navigation/returning-data.md]:
```suggestion
这里我们要怎么做呢？ 使用 [`Navigator.pop`]({{site.api}}/flutter/widgets/Navigator/pop.html) ！
```

File: src/docs/cookbook/navigation/returning-data.md [hunk 3]
```diff
@@ -84,13 +115,19 @@ class SelectionButton extends StatelessWidget {
 
 ## 3. Show the selection screen with two buttons
 
+## 3. 在选择界面显示两个按钮
+
 Now, we'll need to build a selection screen! It will contain two buttons. When
 a user taps on a button, it should close the selection screen and let the home
 screen know which button was tapped!
 
+现在来构建选择界面， 它包含两个按钮，当任意一个按钮被点击的时候，关闭选择页面回退到主屏界面，并让主屏界面知道哪个按钮被点击了。
```
Comment (chenglu) [inline src/docs/cookbook/navigation/returning-data.md]:
```suggestion
现在来构建选择界面，它包含两个按钮，当任意一个按钮被点击的时候，关闭选择页面回退到主屏界面，并让主屏界面知道哪个按钮被点击了。
```

File: src/docs/cookbook/navigation/returning-data.md [hunk 4]
```diff
@@ -131,21 +168,29 @@ class SelectionScreen extends StatelessWidget {
...
 ## 4. When a button is tapped, close the selection screen
 
+## 4. 当任意一个按钮被点击，关闭选择界面回退到主屏界面
+
 Now, we'll want to update the `onPressed` callback for both of our buttons! In
 order to return data to the first screen, we'll need to use the
 [`Navigator.pop`]({{site.api}}/flutter/widgets/Navigator/pop.html)
 method.
 
+接下来我们来更新两个按钮的 `onPressed` 回调函数， 使用 [`Navigator.pop`]({{site.api}}/flutter/widgets/Navigator/pop.html) 回退界面并返回数据给主屏界面。
```
Comment (chenglu) [inline src/docs/cookbook/navigation/returning-data.md]:
```suggestion
接下来我们来更新两个按钮的 `onPressed` 回调函数，使用 [`Navigator.pop`]({{site.api}}/flutter/widgets/Navigator/pop.html) 回退界面并返回数据给主屏界面。
```

File: src/docs/cookbook/navigation/returning-data.md [hunk 5]
```diff
@@ -167,12 +214,18 @@ RaisedButton(
 
 ## 5. Show a snackbar on the home screen with the selection
 
+## 5. 在主屏界面显示 snackbar ，展示选中的项目
```
Comment (chenglu) [inline src/docs/cookbook/navigation/returning-data.md]:
```suggestion
## 5. 在主屏界面显示一个 snackbar，展示选中的项目
```

File: src/docs/cookbook/navigation/returning-data.md [hunk 6]
```diff
@@ -167,12 +214,18 @@ RaisedButton(
...
+
 Now that we're launching a selection screen and awaiting the result, we'll want
 to do something with the information that's returned!
 
+现在，我们跳转到选择界面并等待返回结果， 当结果返回时我们可以做些事情。
```
Comment (chenglu) [inline src/docs/cookbook/navigation/returning-data.md]:
```suggestion
现在，我们跳转到选择界面并等待返回结果，当结果返回时我们可以做些事情。
```

File: src/docs/cookbook/navigation/returning-data.md [hunk 7]
```diff
@@ -167,12 +214,18 @@ RaisedButton(
...
+
 In this case, we'll show a Snackbar displaying the result. To do so, we'll
 update the `_navigateAndDisplaySelection` method in our `SelectionButton`.
 
+在本例中，我们用一个 snackbar 显示结果， 我们来更新 `SelectionButton` 类中的 `_navigateAndDisplaySelection` 方法。
```
Comment (chenglu) [inline src/docs/cookbook/navigation/returning-data.md]:
```suggestion
在本例中，我们用一个 snackbar 显示结果，我们来更新 `SelectionButton` 类中的 `_navigateAndDisplaySelection` 方法。
```

**Comment by songfei** [comment]
@chenglu 感谢修改

**Pull request merged**